src/predictive-text-input/predictive-text-input.tsx

```diff
@@ -250,15 +287,33 @@ export const PredictiveTextInput = <T, V>({
                 $readOnly={readOnly}
                 $error={error}
             >
+                <span
+                    id="predictive-input-instructions"
```

Comment:
do generate a unique id, else the screen reader may refer to the wrong message when the component is rendered more than once in a page